hs_core/views/resource_folder_hierarchy.py

```diff
@@ -126,6 +127,14 @@ def data_store_structure(request):
     )
 
 
+def to_external_url(url):
+    """
+    Convert an internal download file/folder url to the external url.  This should eventually be
+    replaced with with a reverse method that gets the correct mapping.
+    """
+    return url.replace("django_irods/download", "resource", 1)
```

Comment:
What does the final download URL look like? Is there an example?

Reply:
http://192.168.56.102:8000/resource/046bf8a4e4b945498bb856d077797013/data/contents/Outlet-layer1.dbf

The test cases also tell you what it should not look like.

Reply:
@sblack-usu Can you elaborate a bit on why we need to map to external url?

Reply:
This was a request by @dtarb.  This url is exposed to the user when they select a file/folder in a resource and click "get file url".  Because the url is exposed to the user, he wanted the external url to be used.

Reply:
OK. Got it. Thanks for the clarification.